float64 odometer
float32 speed
int32 rc_in1
int32 rc_in2
int32 rc_in3
int32 rc_in4
time timestamp
